code/datums/mind.dm

```diff
@@ -19,6 +19,10 @@
 		yourself.
 */
 
+#define OBJECTIVE_LIST list("assassinate", "blood", "debrain", "protect", "prevent", "brig", "hijack",\
```

Comment:
Would it make sense to use defined here instead of strings so typos get noticed immediately etc? Or even objective types directly.

Reply:
That would be cleaner indeed. I'll see about refactoring it